src/OrderSettings/OrderSettings.php

```diff
@@ -117,7 +117,7 @@ public function getOrder(): Order
     public function getOrderWeight(): int
     {
         if (! $this->orderWeight) {
-            $this->orderWeight = WeightService::convertWeightToGrams($this->order->getTotalWeight());
+            $this->orderWeight = WeightService::convertToGrams($this->order->getTotalWeight(), strtolower(Configuration::get('PS_WEIGHT_UNIT')));
```

Comment:
1. Waarom doe je hier strtolower en op andere plekken niet? 
3. Misschien is het voor dit soort gevallen wel oke (of zelfs "mooi") om de PDK class in je plugin te extenden met zoiets, zodat die parameter overal consistent is.

```php
class WeightService extends \MyParcelNL\Pdk\Service\WeightService {
	public static convertToGrams($amount) {
    return parent::convertToGrams($amount, Configuration::get('PS_WEIGHT_UNIT'));
  }
}
```